src/main.rs

```diff
@@ -1487,7 +1487,7 @@ async fn hgive<'a>(slash_command: LenientForm<SlashCommand>) -> Json<Value> {
         res("Well, I mean ... that's not really anything but ... ok")
     } else {
         let notif_msg = format!(
-            "<@{}> transferred {} {} {}!",
+            "<@{}> transferred <@{}> {} {}!",
```

Comment:
the wording here is awkward, maybe try "<@{}> gave <@{}> {} {}!" instead?

Reply:
Fixed